src/index.ts

```diff
@@ -90,7 +108,7 @@ export interface UploadConfig {
    * that will be used to encrypt the object. Overrides the object metadata's
    * `kms_key_name` value, if any.
    */
-  kmsKeyName?: string;
+  kmsKeyName?: QueryParameters['kmsKeyName'];
```

Comment:
`kmsKeyName` above is a number.  Which one of these is right?

Reply:
String! Thanks!